extensions/amp-audio/validator-amp-audio.protoascii

```diff
@@ -52,6 +52,10 @@ tags: {  # <amp-audio> for AMP (autoplay attribute allowed)
     name: "autoplay"
     value: ""
   }
+  attrs: {
+    name: "preload"
+    value: ""
```

Comment:
Setting `value: ""` means that there can be no attribute value, or the attribute value must be the attribute name.

So in your test examples, validation will fail because it is `preload="auto"`. To make them pass it would have to be either `preload` or `preload="preload"`

Reply:
Ahh then I don't want to specify a value and change it to 

```
attrs: { name: '"preload" }
```

Will that mean that the value can be anything?

Reply:
Yes, that would mean that the value can be anything.

Reply:
Cool - that is what I want then. That way the underlying `<audio>` preload attribute can handle validation. Will change in the latest commit. Thanks for the help!

Reply:
Is there a limited set of possible values? It may be worthwhile to still have that in validation and not just at runtime validation.

`value_regex_casei: "(auto|foo|bar)"`

Reply:
The spec allows three values: auto, none, metadata and an empty string so I will set it to 
```
value_regex_casei: "(auto|none|metadata|)"
```

How does that sound?

Reply:
Minor tweaks, alphabetize and remove last pipe:

Reply:
Does this accept empty string?

Reply:
I believe the trailing `|` was there to allow empty string. We should keep this same as `amp-video`

Reply:
Yup retaining the trailing pipe to allow empty string. This doesn't match `<amp-video>` allowed values exactly, because:
* `<amp-video>` wasn't alphabetized
* <amp-video>` wasn't case insensitive which IMO is better since that is what the underlying implementation in all browsers is as well. 

Does this work for the group?

Reply:
I don't think I've ever seen us use an empty pipe like that but it should work. cc @Gregable in case he can think of a reason not to do it. `value_regex_casei: "(auto|metadata|none|)"`